src/services/AbstractDFG.jl

```diff
@@ -1,3 +1,13 @@
+##==============================================================================
+## AbstractDFG
+##==============================================================================
+##------------------------------------------------------------------------------
+## Broadcasting
+##------------------------------------------------------------------------------
+# to allow stuff like `getFactorType.(dfg, [:x1x2f1;:x10l3f2])`
+# https://docs.julialang.org/en/v1/manual/interfaces/#
+Base.Broadcast.broadcastable(dfg::AbstractDFG) = Ref(dfg)
```

Comment:
oh thats easy, thanks!

Reply:
So you just have to tell julia dfg is a scalar.